src/main/java/seedu/address/ui/ViewCompanyDisplay.java

```diff
@@ -53,10 +55,11 @@ public class ViewCompanyDisplay extends UiPart<Region> {
     /**
      * Displays a company's details in the viewing box.
      */
-    public ViewCompanyDisplay(Company company) {
+    public ViewCompanyDisplay(Company company, String displayString) {
```

Comment:
Does this change the output of the UI? If so, due to the feature freeze, I don't think we're allowed to change the UI output :(